### PR TITLE
Webrtc

### PR DESCRIPTION
**Pull request by halvardssm**

### CODE REVIEW

**Comment by halvardssm** [comment]
Hey @Tschakabaka , could you look through the PR? I think this is fine in regards to the streaming for now until we have completed more of the frontend design. WebRTC and the socket connection should be added with this 💪

**Comment by halvardssm** [comment]
A couple of Components have also been cleaned up so you will see some diff changes with those

File: src/pages/Stream.tsx [hunk 1]
```diff
@@ -11,25 +10,46 @@ export type StreamParams = {
   streamId: string
 }
 
+const tokens = {
+  tokenA:
+    'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VybmFtZSI6InNvY2lhbF9zbGFtX2FkbWluIiwiZXhwIjoxNTkxNzA4NTkxLCJvcmlnSWF0IjoxNTkxNzA4MjkxfQ.Ki9BMu_cKMrXJ9pH7oKvBIM1CkJk7qk9TUGUEvAAwC0',
+  tokenB:
+    'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJicmF2byIsImlhdCI6MTUxNjIzOTAyMn0.n-Fsy8Jx6q9IubgaNZUgooNcsUG_58OVgE9MUTLkMVs',
+}
```
Comment (Tschakabaka) [inline src/pages/Stream.tsx:18]: Are these tokens permanent? If so please add them to a .env file

File: src/services/WSConnection.tsx [hunk 2]
```diff
@@ -1,117 +1,87 @@
...
+  };
 
   ws.onmessage = (msg: MessageEvent) => {
-    const message = JSON.parse(msg.data)
-
-    console.log(message)
-  }
+    const message = JSON.parse(msg.data);
+    console.log(message);
+    const [slammers, connections] = onWsMessage(ws, message);
```
Comment (Tschakabaka) [inline src/services/WSConnection.tsx]: We should settle on our term for the users/artists. I am calling them artists/audience inside of an event and I named my Model plainly **User**

File: frontend/src/components/Connection.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
+import io from 'socket.io-client'
```
Comment (Tschakabaka) [inline frontend/src/components/Connection.js]: Are you still using Socket.io or is everythig solved with simple-peer now?

File: src/pages/Stream.tsx (hunk 1 above)
Comment (halvardssm) [reply src/pages/Stream.tsx:18]: They are just for testing. I can remove them for now

File: src/services/WSConnection.tsx (hunk 2 above)
Comment (halvardssm) [reply src/services/WSConnection.tsx]: You are looking at an outdated version 🤔

File: frontend/src/components/Connection.js (hunk 3 above)
Comment (halvardssm) [reply frontend/src/components/Connection.js]: Same as above. This file no longer exists 🤔

Comment (Tschakabaka) [reply frontend/src/components/Connection.js]: Yep I realized I was looking at commits the wrong way round. I started at the bottom and went up.

**Pull request merged**